scripts/screens/ProfileScreen.py

```diff
@@ -1081,8 +1081,14 @@ def build_profile(self):
                 else:
                     self.profile_elements["talk"].enable()
         elif game.clan.your_cat.moons >= 0 and self.the_cat.ID != game.clan.your_cat.ID and self.the_cat.ID not in game.clan.unknown_cats and not self.the_cat.outside and not game.clan.your_cat.outside:
+            
+            cat_dead_conditions = self.the_cat.dead and (game.clan.your_cat.dead or game.clan.your_cat.skills.meets_skill_requirement(SkillPath.STAR) or game.clan.your_cat.skills.meets_skill_requirement(SkillPath.DARK) or game.clan.your_cat.skills.meets_skill_requirement(SkillPath.GHOST))
+            cat_alive_skills_condition = not self.the_cat.dead and (self.the_cat.skills.meets_skill_requirement(SkillPath.STAR) or self.the_cat.skills.meets_skill_requirement(SkillPath.DARK) or self.the_cat.skills.meets_skill_requirement(SkillPath.GHOST))
+
```

Comment:
these are the old skill checks that allow anyone with a star/df/ghost skill to talk to any dead cat